### PR TITLE
Add AWS RDS commands

### PR DESCRIPTION
**Pull request by satyarohith**
The following commands will be implemented.
- `rds_list` - List RDS instances under your account.

Merge #7 before #8

### CODE REVIEW

File: aws/commands.yaml [hunk 1]
```diff
@@ -16,3 +16,12 @@ commands:
...
       - name: id
+  dynamodb_list:
+    sourcePath: dynamodb/dynamodb_list.js
+    description: List DB Tables under your account.
+    parameters:
+      - name: startTable
+        optional: true
+  rds_list:
+    sourcePath: rds/rds_list.js
+    description: List RDS instances under your account.
```
Comment (niks3089) [inline aws/commands.yaml]: Expand RDS for people not knowing what it means

File: aws/dynamodb/dynamodb_list.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ * @return {Promise<SlackBodyType>} Response body
+ */
+async function _command(params, commandText, secrets = {}) {
+  const {awsAccessKey, awsSecretKey, awsRegion} = secrets;
+
+  if (!awsAccessKey || !awsSecretKey || !awsRegion) {
+    return {
+      response_type: 'ephemeral',
+      text:
+        'You must create secrets for `awsAccessKey`, `awsSecretKey` and `awsRegion` to use this command '
```
Comment (niks3089) [inline aws/dynamodb/dynamodb_list.js]: Need to add /nc secret info here.

Comment (satyarohith) [reply aws/dynamodb/dynamodb_list.js]: Fixed in #7

File: aws/commands.yaml (hunk 1 above)
Comment (satyarohith) [reply aws/commands.yaml]: Updated!

**Review by niks3089 — APPROVED** [review]

**Pull request merged**